packages/docs/site/docs/09-blueprints-api/03-data-format.md

```diff
@@ -34,34 +30,90 @@ import BlueprintExample from '@site/src/components/Blueprints/BlueprintExample.m
 
 ```js
 {
 	"$schema": "https://playground.wordpress.net/blueprint-schema.json",
-	"landingPage": "/wp-admin/",
-	// ...
 }
 ```
 
-## Preferred Versions
+## Landing page
 
-The `preferredVersions` property, unsurprisingly, declares the preferred of PHP and WordPress versions to use. It can contain the following properties:
+The `landingPage` property tells Playground which URL to navigate to after the Blueprint has been run. This is a great tool, especially when creating theme or plugin demos. Often, you will want to start Playground in the Site Editor or have a specific post open in the Post Editor. Make sure you use a relative path. 
 
--   `php` (string): The preferred PHP version to use. Defaults to "latest". Only accepts major versions, like "7.4" or "8.0". Minor versions like "7.4.1" are not supported.
--   `wp` (string): Loads the specified WordPress version. Supported values: The last three major WordPress versions—minor versions, like `6.5.1`, are not supported. As of April 4, 2024, that's `6.3`, `6.4`, `6.5`. You can also use these values: `latest` (default), `nightly`, or `beta`.
+```js
+{
+	"landingPage": "/wp-admin/site-editor.php",
+}
+```
+
+## Preferred versions
+
+The `preferredVersions` property declares your preferred PHP and WordPress versions. It can contain the following properties:
+
+-   `php` (string): Loads the specified PHP version. Accepts `7.0`, `7.1`, `7.2`, `7.3`, `7.4`, `8.0`, `8.1`, `8.2`, `8.3`, or `latest`. Minor versions like `7.4.1` are not supported.
+-   `wp` (string): Loads the specified WordPress version. Accepts the last three major WordPress versions. As of June 1, 2024, that's `6.3`, `6.4`, or `6.5`. You can also use the generic values `latest`, `nightly`, or `beta`.
+
+```js
+{
+	"preferredVersions": {
+		"php": "8.0",
+		"wp": "6.5"
+	},
+}
+```
 
 ## PHP extensions
 
-The `phpExtensionBundles` property is an array of PHP extension bundles to load. The following bundles are supported:
+The `phpExtensionBundles` property is an array of PHP extension bundles that should be loaded. The following bundles are supported:
 
--   `light`: Default choice. Saves 6MB of downloads, loads none of the extensions below.
--   `kitchen-sink`: Installs [`gd`](https://www.php.net/manual/en/book.image.php), [`mbstring`](https://www.php.net/manual/en/mbstring.installation.php), [`iconv`](https://www.php.net/manual/en/function.iconv.php), [`openssl`](https://www.php.net/manual/en/book.openssl.php), [`libxml`](https://www.php.net/manual/en/book.libxml.php), [`xml`](https://www.php.net/manual/en/xml.installation.php), [`dom`](https://www.php.net/manual/en/intro.dom.php), [`simplexml`](https://www.php.net/manual/en/book.simplexml.php), [`xmlreader`](https://www.php.net/manual/en/book.xmlreader.php), [`xmlwriter`](https://www.php.net/manual/en/book.xmlwriter.php)
+-   `light`: Default choice. It saves 6MB of downloads and loads none of the extensions below.
```

Comment:
`kitchen-sink` is now the default.

It looks like default was updated in the docs on Apr 3rd [here](https://github.com/WordPress/wordpress-playground/commit/b8686b9217dbdc6e418cf2999451edf042220a5a#diff-eccc9c78da07b82ab21cc870427dbd045cceae6edb666c4370d886eda97de5a1R60) but was mistakenly reverted [here](https://github.com/WordPress/wordpress-playground/commit/886e91472f5ac3ef4e65a6dd67dcbda408167d1e#diff-eccc9c78da07b82ab21cc870427dbd045cceae6edb666c4370d886eda97de5a1R60) (cc @flexseth).